accel-mma84/src/lib.rs

```diff
@@ -0,0 +1,138 @@
+#![feature(alloc_system)]
+extern crate alloc_system;
+
+extern crate tessel;
+
+use std::io;
+
+#[repr(u8)]
+pub enum ScaleRange {
+    Scale2G = 0x00,
+    Scale4G = 0x01,
+    Scale8G = 0x10,
```

Comment:
As hexadecimal integer literals, these are incorrect. The values above are: 0, 1, 16. The scale values for this chip are defined on page 22 here: https://cdn.sparkfun.com/datasheets/Sensors/Accelerometers/MMA8452Q-rev8.1.pdf 

I specifically call out that they are wrong as hex literals, because if the 0x is changed to 0b, then they are all correct. Presently, 2G and 4G will work correctly, but 8G will not, since the value is 16, instead of 2. Reference implementation: https://github.com/rwaldron/johnny-five/blob/master/lib/accelerometer.js#L286-L566 specially line 329